lib/rules/lines-between-class-members.js

```diff
@@ -120,20 +93,26 @@ module.exports = {
                     const curFirst = sourceCode.getFirstToken(body[i]);
                     const curLast = sourceCode.getLastToken(body[i]);
                     const nextFirst = sourceCode.getFirstToken(body[i + 1]);
-                    const isPadded = isPaddingBetweenTokens(curLast, nextFirst);
                     const isMulti = !astUtils.isTokenOnSameLine(curFirst, curLast);
                     const skip = !isMulti && options[1].exceptAfterSingleLine;
-
+                    const beforePadding = findLastConsecutiveTokenAfter(curLast, nextFirst, 1);
+                    const afterPadding = findFirstConsecutiveTokenBefore(nextFirst, curLast, 1);
+                    const isPadded = afterPadding.loc.start.line - beforePadding.loc.start.line > 1;
+                    const hasPaddedComments = sourceCode.commentsExistBetween(beforePadding, afterPadding);
```

Comment:
I think we should check for any tokens between, not just comments.

For example, fix for the following code will remove the semicolon:

```js
/* eslint lines-between-class-members: ["error", "never"] */

class A {
    foo() {}

    ;

    bar() {}
}
```